src/construction/construction.py

```diff
@@ -51,24 +54,19 @@ def run_construction(  # noqa: C901
         construction_logger.info("Skipping Construction...")
         return snapshot_df
 
-    # Check the construction file exists and has records, then read it
+    # Obtain construction paths
     network_or_hdfs = config["global"]["network_or_hdfs"]
```

Comment:
If you pull in develop with the branch you reviewed for me, you can have 
construction_config = config["construction_paths"']